src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -280,7 +280,7 @@ def __call__(
             uncond_embeddings = self.text_encoder(uncond_input.input_ids.to(self.device))[0]
 
             # duplicate unconditional embeddings for each generation per prompt
-            uncond_embeddings = uncond_embeddings.repeat_interleave(batch_size * num_images_per_prompt, dim=0)
+            uncond_embeddings = uncond_embeddings.repeat_interleave(num_images_per_prompt, dim=0)
```

Comment:
`repeat_interleave` is not compatible with some devices (`mps`, possibly ONNX). Can we do this the same way as the others?

```suggestion
            seq_len = uncond_embeddings.shape[1]
            uncond_embeddings = uncond_embeddings.repeat(1, num_images_per_prompt, 1)
            uncond_embeddings = uncond_embeddings.view(batch_size * num_images_per_prompt, seq_len, -1)
```

Reply:
Also i think we dont need view here, just repeat batch axis, right?

Reply:
Yes, you are right, I blindly copied the lines.

Reply:
Actually, I just tested locally and we do need to do the reshape.

@hadaev8 We want to merge this for today's release as it's a good fix (we'll do the tests later). I'll make these final changes in your PR if you don't mind and we should be set :)